packages/ui/resolution/tests/resolution-specs.js

```diff
@@ -85,12 +88,22 @@ describe('Resolution Specs', () => {
 			expect(getScreenType({width: HD.width + 1, height: XGA.height - 1})).toBe('hd');
 			expect(getScreenType({width: HD.width + 1, height: XGA.height + 1})).toBe('hd');
 
+			expect(getScreenType({width: UWHD.width - 1, height: UWHD.height - 1})).toBe('hd');
+			expect(getScreenType({width: UWHD.width + 1, height: UWHD.height - 1})).toBe('hd');
+			expect(getScreenType({width: UWHD.width - 1, height: UWHD.height + 1})).toBe('hd');
+
+			expect(getScreenType({width: UWUXGA.width - 1, height: UWHD.height - 1})).toBe('hd');
+			expect(getScreenType({width: UWUXGA.width + 1, height: UWHD.height - 1})).toBe('hd');
```

Comment:
can you put in the PR description why these hybrid scenarios are needed? UWUXGA+ UWHD dimensions are used